docs/terms/data_assistant.md

```diff
@@ -0,0 +1,81 @@
+
+### Relationships to other objects
+
+A Data Assistant implements a pre-configured <TechnicalTag tag="profiler" text="Rule Based Profiler" /> in order to gather Metrics and propose an Expectation Suite based on the introspection of the Batch or Batches contained in a provided <TechnicalTag tag="batch_request" text="Batch Request" />.
+
+## Use cases
+
+<CreateHeader/>
+
+Data Assistants are an ideal starting point for creating your Expectations.  If you are working with data that you are not familiar with, a Data Assistant can give you an overview by introspecting it and generating a series of relevant Expectations using estimated parameters for you to review. If you use the `"flag_outliers"` value for the `estimation` parameter your generated Expectations will have parameters that disregard values that the Data Assistant identifies as outliers. Using the Data Assistant's `plot_metrics()` method will then give you a graphical representation of the generated Expectations.  This will further assist you in spotting outliers in your data when reviewing the Data Assistant's results.
+
+Even when working with data that you are familiar with and know is good, a Data Assistant can use the `"exact"` value for the `estimation` parameter to provide comprehensive Expectations that exactly reflect the values found in the provided data.
+
+## Features
+
+### Easy profiling
+
+Data Assistants can function as pre-configured Profilers.  Since you can access them directly from your Data Context, they are easily accessible.  This ensures that they will always provide a quick, simple entry point to creating Expectations and <TechnicalTag tag="profiling" text="Profiling" /> your data.  However, the rules implemented by a Data Assistant are fully exposed in the parameters for its `run(...)` method.  This means that while you can use a Data Assistant easily out of the box, you can also customize it behavior to take advantage of the domain knowledge possessed by subject-matter experts.
+
+### Multi-Batch introspection
+
+Data Assistants really show their worth when you provide them a Batch Request that contains multiple Batches.  With previous Profilers you would only be able to introspect a single Batch at a time.  This meant that the Expectation Suite generated would only reflect a single Batch.  If you had many Batches of data that you wanted to build inter-related Expectations for, you would have needed to run each Batch individually and then manually compare and update the Expectation parameters that were generated.  With a Data Assistant, that process is automated.  You can provide a Data Assistant multiple Batches and get back Expectations that have parameters based on, for instance, the mean or median value of a column on a per-Batch basis. 
```

Comment:
@Rachel-Reverie This is a bit more nuanced -- happy to discuss to offer a perspective.  Thanks!